### ISSUE TITLE
TEE_GetSystemTime invalid time

### ISSUE THREAD

**Issue by jdeus** [OP-TEE/build#315]
On rpi3 with the default governor (powersave), The results of _seconds_  of a TEE_Time passed to TEE_GetSystemTime must be divided by 10 to obtain a valid result.

**Example:**

> TEE_Time startt, stopt;
> 
> TEE_GetSystemTime(&startt);
> (...)
> TEE_GetSystemTime(&stopt);
> 
> IMSG("%d.%d", startt.seconds, startt.millis);
> IMSG("%d.%d", stopt.seconds, stopt.millis);

**Produces:**

> I/TA:  2831.571
> I/TA:  2929.64

While only ~9 seconds have elapsed.

**Expected:**

> I/TA:  283.1571
> I/TA:  292.964

**Comment by jbech-linaro** [comment]
Yes, this seems like a bug to me. Do you think you could make a patch for it?
